build-system/runner/src/org/ampproject/AmpPass.java

```diff
@@ -85,12 +79,6 @@ public AmpPass(AbstractCompiler compiler, boolean isProd,
     } else if (isProd && isFunctionInvokeAndPropAccess(n, "$mode.getMode",
         ImmutableSet.of("minified"))) {
       replaceWithBooleanExpression(true, n, parent);
-    } else {
-      if (isProd) {
-        maybeReplaceRValueInVar(n, prodAssignmentReplacements);
-      }
-      maybeReplaceRValueInVar(n, assignmentReplacements);
-      maybeReplaceCallWithVersion(n, parent);
```

Comment:
Is there no default case?

Reply:
the default case is now handled by babel